zerver/forms.py

```diff
@@ -315,12 +315,14 @@ def clean(self) -> Dict[str, Any]:
                 # We exclude mirror dummy accounts here. They should be treated as the
                 # user never having had an account, so we let them fall through to the
                 # normal invalid_login case below.
-                raise ValidationError(mark_safe(DEACTIVATED_ACCOUNT_ERROR))
+                raise ValidationError(mark_safe(DEACTIVATED_ACCOUNT_ERROR),
+                                      params={'username': username})
```

Comment:
Does this `mark_safe`  interaction do the right thing if there's some HTML in the username?  

Also, can you post a screenshot showing this working?